core/micro-services/workflow-operator/src/main/scala/edu/uci/ics/amber/operator/common/Utils.scala

```diff
@@ -0,0 +1,28 @@
+package edu.uci.ics.amber.operator.common
```

Comment:
I suggest we do not have `common` in package names. it has collisions with `core` and `utils` in its meaning. Could we make this a `util` package? (also, not using plural form)

Reply:
I remove the common package. As there is only `metadata` under `common`, I made metadata as `operator.metadata`